scripts/upgrade

```diff
@@ -105,30 +134,42 @@ ynh_add_systemd_config
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
+ynh_script_progression --message="Modifying a config file..."
+
+ynh_add_config --template="../conf/gunicorn.py" --destination="$final_path/gunicorn.py"
+ynh_add_config --template="../conf/settings.py" --destination="$final_path/settings.py"
 
+for price in $(echo $prices | sed "s/,/ /"); do
+    frequency=$(echo $price | cut -d/ -f1)
+    currency=$(echo $price | cut -d/ -f2)
+    price_id=$(echo $price | cut -d/ -f3)
+    echo "DONATION['$frequency']['$currency'] = '$price_id'" >> "$final_path/settings.py"
```

Comment:
Not the biggest issue ever, but because we're modifying the settings.py after `ynh_add_config`, this fucks up the checksum (also in the install script)

I was thinking "let's just define the vars prior to calling `ynh_add_config`, but I understand it's because there's a for loop ...

Yet one thing could be to have that for loop *inside* the settings.py, considering it's a python script and doing a `split()` (equivalent to `cut`) is perfectly doable

Othersolution is to just call `ynh_store_checksum_bla` but hmpf

Reply:
Just added `ynh_store_file_checksum --file="$final_path/settings.py"` during install and upagrde